pkg/plugin/v3/scaffolds/internal/templates/main.go

```diff
@@ -213,13 +214,17 @@ func init() {
 }
 
 func main() {
+{{- if not .ComponentConfig }}
 	var metricsAddr string
 	var enableLeaderElection bool
 	flag.StringVar(&metricsAddr, "metrics-addr", ":8080", "The address the metric endpoint binds to.")
 	flag.BoolVar(&enableLeaderElection, "enable-leader-election", false,
 		"Enable leader election for controller manager. " +
 		"Enabling this will ensure there is only one active controller manager.")
-
```

Comment:
we can add here 
// nolint:lll

To fix it

Reply:
Thanks, the flag message was actually wrong so I'm glad this caught it. Updating now.